### ISSUE TITLE
Can't submit quest "Miner's Fortune" in Ratchet

### ISSUE THREAD

**Issue by Goldk4t** [chromiecraft/chromiecraft#3200]
### What client do you play on?

enUS

### Faction

- [ ] Alliance
- [X] Horde

### Content Phase:

- [ ] Generic
- [X] 1-19
- [ ] 20-29
- [ ] 30-39
- [ ] 40-49
- [ ] 50-59
- [ ] 60

### Current Behaviour

Quest: [Miner's Fortune](https://wowgaming.altervista.org/aowow/?quest=896)
NPC: [Wharfmaster Dizzywig](https://wowgaming.altervista.org/aowow/?npc=3453)
Item: [Cats Eye Emerald](https://wowgaming.altervista.org/aowow/?item=5097)

I couldn't talk to Wharfmaster Dizzywig, but i had the quest item in my inventory, it was marked as (complete) and the map showed the position of the npc to submit the quest.

Also i didn't noticed the quest notification when i looted the item, but maybe i just passed it.

Here are some screenshots:
![minersfortunebug](https://user-images.githubusercontent.com/74498144/158067406-bf8fa3d1-18a5-4581-b665-7bd9ba6a6345.png)
![minersfortunebug2](https://user-images.githubusercontent.com/74498144/158067408-bbf57077-ff5c-409f-b951-b5d2da919d0c.png)

### Expected Blizzlike Behaviour

Self-explaining

### Steps to reproduce the problem

1. Accept quest
2. Farm the questitem
3. Return to ratchet and try to submit the quest

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/a775ddcb5b935be3358644be27df282fc29a9936

### Operating system

Windows 10

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [mod-save-inventory](https://github.com/Nyeriah/mod-save-inventory)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-exchange-npc](https://github.com/55Honey/Acore_ExchangeNpc)
- [lua-custom-worldboss](https://github.com/55Honey/Acore_CustomWorldboss)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)
- [lua-zone-debuff](https://github.com/55Honey/Acore_ZoneDebuff)

### Customizations

None

### Server

ChromieCraft

**Comment by Dejanos** [comment]
Just tested the quest on the PTR, took quest, farmed for the quest item, got quest item, came back to the quest giver, could talk to him. 
After that I abandoned quest to see if I could talk to him with the quest item still in my bag, but quest item was gone on abandon so I added the quest item to my inventory and took the quest again and I could talk to him. Here is a SS.
![WoWScrnShot_031322_181203](https://user-images.githubusercontent.com/100415626/158071106-f47c2200-8671-49d5-adb6-b29f6f70d01b.jpg)
![quest complete](https://user-images.githubusercontent.com/100415626/158071138-d0d627e1-7c4a-4079-8e6b-15c1f6b2e490.png)
